InvenTree/InvenTree/models.py

```diff
@@ -73,19 +72,16 @@ def getUniqueChildren(self, unique=None):
 
         unique.add(self.id)
 
+        # commenting this out as it throws errors.
```

Comment:
What errors are you getting here? This code is required to ensure that a category cannot be set as a parent or child of itself.